site/content/docs/FAQs/_index.md

```diff
@@ -35,7 +37,7 @@ See the [repository help docs](../repositories) for more information.
 
 #### What is a Policy?
 
-A `policy` is a set of rules that tells Kopia how to create/manage snapshots; this includes features such as [compression, snapshot retention, and scheduling when to take automatically snapshots](../features#policies-control-what-and-how-filesdirectories-are-saved-in-snapshots).
+A `policy` is a set of rules that tells Kopia how to create/manage snapshots; this includes features such as [compression, snapshot retention, and scheduling when to take snapshots automatically](../features#policies-control-what-and-how-filesdirectories-are-saved-in-snapshots).
```

Comment:
That is fine, or alternatively:

```suggestion
A `policy` is a set of rules that tells Kopia how to create/manage snapshots; this includes features such as [compression, snapshot retention, and scheduling when to automatically take snapshots](../features#policies-control-what-and-how-filesdirectories-are-saved-in-snapshots).
```

Reply:
Thanks for the feedback. I would leave it as it is, i guess.